testing/py-pympler/APKBUILD

```diff
@@ -0,0 +1,52 @@
+# Contributor: Orson Teodoro <orsonteodoro@hotmail.com>
+# Maintainer: Orson Teodoro <orsonteodoro@hotmail.com>
+
+pkgname=py-pympler
+_pkgname=pympler
+__pkgname=Pympler
+pkgver=0.5
+pkgrel=0
+pkgdesc="A development tool to measure, monitor and analyze the memory behavior\
+ of Python objects."
+url="http://pythonhosted.org/Pympler/"
+arch="noarch"
+license="MIT"
+makedepends="py-setuptools py3-setuptools"
+source="$pkgname-$pkgver.tar.gz::https://pypi.python.org/packages/9e/4b/04c8338a9a2a22645ada1d52c59df2b81d644400f6cca10f63072e5f60cd/$__pkgname-$pkgver.tar.gz#md5=831d12beca2ff2ea56407c5ed8ffc4b0"
```

Comment:
Im not sure I like this source url. What needs to be done when updating to new version? Its not just to change pkgver apparently. Would be nice with a comment explaining what to do.